### ISSUE TITLE
pbts: Basic proto3 generation errors 

### ISSUE THREAD

**Issue by nizsheanez** [dcodeIO/protobuf.js#577]
protobuf.js version: 6.2.1
but it works with protobuf.js version: 6.1.1

Protobuf 3 file: 
```
syntax = "proto3";

message SearchRequest {
  string query = 1;
  int32 page_number = 2;
  int32 result_per_page = 3;
}
```

Commands i run: 
```bash
./node_modules/.bin/pbjs -t static-module ./helloworld.proto > hello.js
./node_modules/.bin/pbts --name alex hello.js
```

Error1:
```
node_modules/protobufjs/lib/tsd-jsdoc/publish.js:422
        if (/^Object\b/i.test(element.type.names[0]) && element.properties) {
                                          ^

TypeError: Cannot read property 'names' of undefined
    at handleMember (/node_modules/protobufjs/lib/tsd-jsdoc/publish.js:422:43)
    at handleElement (/node_modules/protobufjs/lib/tsd-jsdoc/publish.js:302:20)
    at /node_modules/protobufjs/lib/tsd-jsdoc/publish.js:370:13
    at Array.forEach (native)
    at handleClass (node_modules/protobufjs/lib/tsd-jsdoc/publish.js:366:28)
    at handleElement (node_modules/protobufjs/lib/tsd-jsdoc/publish.js:295:16)
    at Array.forEach (native)
    at Object.publish (node_modules/protobufjs/lib/tsd-jsdoc/publish.js:78:34)
    at Object.module.exports.cli.generateDocs (node_modules/protobufjs/node_modules/jsdoc/cli.js:430:39)
    at Object.module.exports.cli.processParseResults (node_modules/protobufjs/node_modules/jsdoc/cli.js:383:20)
    at module.exports.cli.main (node_modules/protobufjs/node_modules/jsdoc/cli.js:227:14)
    at Object.module.exports.cli.runCommand (/node_modules/protobufjs/node_modules/jsdoc/cli.js:180:5)
    at /node_modules/protobufjs/node_modules/jsdoc/jsdoc.js:103:9
    at Object.<anonymous> (/node_modules/protobufjs/node_modules/jsdoc/jsdoc.js:104:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
```

Error2: 
```
    /**
     * Verifies a SearchRequest.
     * @param {SearchRequest|Object} message SearchRequest or plain object to verify
     * @returns {?string} `null` if valid, otherwise the reason why it is not
     */
    static verify: /node_modules/protobufjs/cli/pbts.js:83
                throw err;
                ^

Error: code 1
    at ChildProcess.<anonymous> (/node_modules/protobufjs/cli/pbts.js:79:23)
    at emitTwo (events.js:111:20)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:885:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:501:12)
```

**Comment by mmikulicic** [comment]
I noticed that latest master (6.3.0) doesn't exhibit this problem

@dcodeIO would it be possible to have that version published ?

**Comment by dcodeIO** [comment]
Going to release once the changelog stuff is ready. Hopefully, soon™

**Comment by dcodeIO** [comment]
Released as 6.3.0 on npm meanwhile. Feel free to reopen the issue if there are any issues!
